test/unit/HeartbeatTest.js

```diff
@@ -0,0 +1,36 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const index = require('../../index')
+
+// Configure Chai
+use(sinonChai)
+
+describe('HeartbeatTest.js unit tests', () => {
+  describe('when given state transitions array', () => {
+    it('should reformat the state transitions array', () => {
+      const stateTransitionsArray1 = [[0, 0, 1]]
+      const stateTransitionsArrayReadable1 = [
+        {
+          state: 0,
```

Comment:
What does state 0 mean?

Reply:
0 is referring to the idle state, and correspond to the [state diagram](https://docs.google.com/drawings/d/1sq7MucmDWfG4PBPH-IGz7HAvKjXkQ7c1LccWlS7bQo8/edit). Didn't cross my mind that state numbers would be unclear - I'll use the function to replace the numbers with the state names.